dependency/actions/compile/action.yml

```diff
@@ -0,0 +1,32 @@
+name: 'Compile PHP on Target'
```

Comment:
Can you add a README to the `actions/compile` directory about how to run locally?

Reply:
> also a bit of context about how this code is set up would probably be useful - the extensions-manifests and binary-builder directories are atypical obviously

This is sort of part of the confusion as it requires a decent understanding of the Ruby code (ie MiniPortile). If I gain any context, I'll definitely add it.

Reply:
Ok. no need to block this PR on this request